rdbms/src/main/java/io/spine/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -212,12 +226,31 @@ public Builder setDataSource(DataSourceConfig dataSource) {
         }
 
         /**
+         * Sets {@link TypeMapping} to use for working with the database.
```

Comment:
`to use for working with the database` should be rephrased.

Reply:
Done.